docs/coding-style.md

```diff
@@ -22,6 +22,16 @@ Ordering of class members should be the following:
 
 Furthermore, each of these categories should be ordered from higher to lower accessibility level (public, internal, protected, private).
 
+Static fields and properties should be placed before instance ones. 
+
+Static methods are preferred to be after instance methods.
+
+## Naming conventions
+
+Single variable lambdas should use `x` as the variable name (based on lambda calculus λx). Multi variable lambdas should use descriptive names, where `x` can be used for the main iterated item like `(x, index) => ...`.
+
+Variable name `sut` (System Under Test) is allowed in unit tests that really tests a single unit (contrary to our usual rule integration unit tests).
```

Comment:
Instead of `allowed` I would say `recommended`.

Reply:
(I hijacked your comment to do the change in GitHub UI 😄 )